molgenis-security/src/main/java/org/molgenis/security/MolgenisWebAppSecurityConfig.java

```diff
@@ -137,6 +140,10 @@ protected void configure(HttpSecurity http) throws Exception {
     requestCache.setRequestMatcher(createWebRequestMatcher());
     http.requestCache().requestCache(requestCache);
 
+    http.sessionManagement()
+        .maximumSessions(MAXIMUM_CONCURRENT_USER_SESSIONS)
```

Comment:
That'll break concurrent use by functional / admin/admin accounts.

Reply:
was wondering about that , why would it break ?

Reply:
switched to using molgenis SecurityContextRegistry, removes need to set max sessions